app/views/feeds/_form_main.html.erb

```diff
@@ -0,0 +1,203 @@
+<div class="col-12 mb-4 prx-field-group">
+  <div class="form-floating">
+    <%= form.text_field :feed_link, value: feed.published_url, disabled: true %>
+    <%= form.label :feed_link %>
+  </div>
+
+  <%= link_to(feed.published_url, class: "prx-btn-help", target: :_blank) do %>
+    <span class="material-icons">link</span>
+  <% end %>
+</div>
+
+<div class="col-12 mb-4 prx-field-group">
+  <div class="form-floating">
+    <%= form.text_field :file_name %>
+    <%= form.label :file_name %>
+  </div>
+
+  <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.file_name") %>"><span class="material-icons">help</span></button>
+</div>
+
+<% unless feed.default? %>
+  <div class="col-12">
+    <div class="mb-4 prx-field-group">
+      <div class="form-floating">
+        <%= form.text_field :slug %>
+        <%= form.label :slug %>
+      </div>
+
+      <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.slug") %>"><span class="material-icons">help</span></button>
+    </div>
+  </div>
+
+  <div class="col-12">
+    <hr>
+    <h3><%= t(".headers.overrides") %></h3>
+    <p><%= t(".help.overrides") %></p>
+    <div class="mb-4">
+      <div class="form-floating">
+        <%= form.text_field :title %>
+        <%= form.label :title %>
+      </div>
+    </div>
+
+    <div class="mb-4 prx-field-group">
+      <div class="form-floating">
+        <%= form.text_field :subtitle %>
+        <%= form.label :subtitle %>
+      </div>
+
+      <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.subtitle") %>"><span class="material-icons">help</span></button>
+    </div>
+
+    <div class="mb-4 prx-field-group">
+      <div class="form-floating">
+        <%= form.text_area :description %>
+        <%= form.label :description %>
+      </div>
+
+      <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.description") %>"><span class="material-icons">help</span></button>
+    </div>
+
+    <div class="mb-4 prx-field-group">
+      <div class="form-floating">
+        <%= form.text_area :summary %>
+        <%= form.label :summary %>
+      </div>
+
+      <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.summary") %>"><span class="material-icons">help</span></button>
+    </div>
+  </div>
+
+  <div class="col-12">
+    <hr>
+    <h3><%= t(".headers.private") %></h3>
+    <div class="prx-field-group">
+      <div>
+        <%= form.check_box :private %>
+        <%= form.label :private %>
```

Comment:
TODO: link this check box to displaying the below Auth Tokens partial using turbo